README.md

```diff
@@ -71,6 +71,8 @@ This is the recommended method of starting the blockchain services, with the aux
 Before attempting the start up please ensure that you are not running any services that are listening on the following TCP ports: 9656, 7434, 5000, 8545, 5432, 5433.
 All commands should be run from the root of the repo.
 
+**NOTE** known to work with `docker-compose version 1.24.0, build 0aa59064`, version `1.17` has had problems
```

Comment:
Can you note that this is for Linux please? I'm on `Docker version 18.09.2, build 6247962` on Mac and this executes successfully for me.

Reply:
> Can you note that this is for Linux please?

hm, is it? (note it is about `docker-compose` not `docker` version - can you check your `docker-compose --version`? Ino has `1.24` as well).

Reply:
Argh, `docker --version` vs `docker-compose --version` :(